Project: Building Your Personal Website: Create Assignment section to fix rule TOP004 violation (#30289)
TheOdinProject/curriculum: getting_hired/preparing_for_job_search/project_portfolio.md

### Show designs

A list of portfolios of professional developers. Students should analyze these sites to understand what should be on their portfolios and how it should be presented.

<details markdown="block">

  <summary> Links for designs </summary>

- [Stratis Bakas](https://stratisbakas.com/)
- [Matt Farley](https://mattfarley.ca/)
- [Dejan Markovic](https://www.dejan.works/)
- [Ian Lunn](https://ianlunn.co.uk/)
- [Ben Adam](https://benadam.me/)
- [Seb Kay](https://sebkay.com/)
- [Andriy Chemerynskiy](https://andrewchmr.com/)
- [Chris Ota's Portfolio](https://www.otadesigns.com/)
- [Pierre Nel](https://pierre.io/)
- [Adrien Laurent's Portfolio](https://adrienlaurent.fr/)
- [Thomas' Portfolio](https://www.thomasbosc.com)
- [Timmy O’Mahony](https://timmyomahony.com/)
- [Elliot's Portfolio](https://elliotcondon.com/)
- [James Warner](https://jmswrnr.com/)
- [Tiago Sá's Portfolio](https://i-am-tiago.com/)
- [Patrick David](https://bepatrickdavid.com/)
- [Luis Krötz](https://luiskr.com/)

</details>

### Assignment

<div class="lesson-content__panel" markdown="1">

> 1. Read this Forbes article [Why Every Job Seeker Should Have a Personal Website](http://www.forbes.com/sites/jacquelynsmith/2013/04/26/why-every-job-seeker-should-have-a-personal-website-and-what-it-should-include/) to understand why having a personal website can boost your job search.

Reviewer: Nittiest of nits
```suggestion
1. Read about [Why Every Job Seeker Should Have a Personal Website](http://www.forbes.com/sites/jacquelynsmith/2013/04/26/why-every-job-seeker-should-have-a-personal-website-and-what-it-should-include/) to understand why having a personal website can boost your job search.
```
